plugin/trino-exchange/src/main/java/io/trino/plugin/exchange/FileSystemExchangeConfig.java

```diff
@@ -99,6 +101,20 @@ public FileSystemExchangeConfig setExchangeSinkBuffersPerPartition(int exchangeS
         return this;
     }
 
+    @NotNull
+    public DataSize getExchangeSinkMaxFileSize()
+    {
+        return exchangeSinkMaxFileSize;
+    }
+
+    @Config("exchange.sink-max-file-size")
```

Comment:
I would suggest using softer naming: `sink-target-file-size`. Technically we can overshoot the size if a single row is larger than the configured value. "target" naming also allows for more elastic tweaks in implementation without mismanaging user expectations.
If you change name here update also in-code variable names.